src/field.rs

```diff
@@ -100,7 +100,7 @@ impl<F: Field> Element<F> {
         }
     }
 
-    pub fn exp(&self, power: &Self) -> Self {
+    pub fn exponentiation(&self, power: &Self) -> Self {
```

Comment:
Renamed to `exponentiation` for notation consistency away from references to `Expression`.